DTS-STN/canadian-dental-care-plan
# AB#3236 -- Updated the federal/provincial programs json with new data

### Description
Updated federal/provincial benefit lists with new json data.

### Related Azure Boards Work Items
[AB#3236](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3236)

### Screenshots (if applicable)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/155585182/733c5d97-8e11-435c-adbd-885378a8ee64)

### Checklist

### Test Instructions
Go to the federal-provincial-territorial-benefits page to confirm updated options.

### Additional Notes
n/a
